client/ayon_houdini/plugins/publish/validate_export_is_a_single_frame.py

```diff
@@ -48,12 +48,12 @@ def get_invalid(cls, instance):
             return
 
         if frame_start != frame_end:
-            invalid.append(instance.data["instance_node"])
+            rop = hou.node(instance.data["instance_node"])
             cls.log.error(
                 "Invalid frame range on '%s'."
                 "You should use the same frame number for 'f1' "
                 "and 'f2' parameters.",
-                instance.data["instance_node"].path()
+                rop.path()
             )
 
-        return invalid
+        return [rop]
```

Comment:
You dont need to remove invalid variable as you can use `invalid` to append `instance.data["instance_node"]` value and  then return `invalid` to solve the variable issue found in the requested changes I showed

Reply:
my bad. I didn't notice the indentation.
https://github.com/ynput/ayon-houdini/pull/202/commits/6caa2c826e42e17b09d80f2c626fc05d1ffd4119